src/main/java/services/notification_sending/EmailNotificationPresenter.java

```diff
@@ -0,0 +1,81 @@
+package services.notification_sending;
+
+import java.util.Properties;
+import javax.mail.*;
+import javax.mail.internet.*;
```

Comment:
Change wildcard imports to explicit. You can update your IDE settings and make the auto bundling count like 999